addons/mail/static/src/js/models/threads/thread.js

```diff
@@ -304,7 +304,7 @@ var Thread = AbstractThread.extend(ServicesMixin, {
     _generateEmojis: function (htmlString) {
         _.each(emojis, function (emoji) {
             _.each(emoji.sources, function (source) {
-                var escapedSource = String(source).replace(/([.*+?=^!:${}()|[\]/\\])/g, '\\$1');
+                var escapedSource = _.escape(String(source)).replace(/([.*+?=^!:${}()|[\]/\\])/g, '\\$1');
```

Comment:
And hopefully this can be reverted if using the correct escape function at the correct time?